parameters/world_modeling/world_model.yaml

```diff
@@ -84,9 +84,9 @@ plugins:
       initial_pose_topic: /amigo/initialpose
       num_particles: 500
       initial_pose:
-          x: -1.0 #$(env ROBOT_INIT_X)
-          y: 0.0 #$(env ROBOT_INIT_Y)
-          rz: 0.0 #$(env ROBOT_INIT_PHI)
+          x: $(env ROBOT_INIT_X 0.0)
+          y: $(env ROBOT_INIT_Y 0.0)
+          rz: $(env ROBOT_INIT_PHI 0.0)
```

Comment:
No idea what this means, don't think native yaml has replace magics.

Reply:
Nope it is is tue_config.